src/index.ts

```diff
@@ -66,12 +68,19 @@ const ifs = unixfs(helia)
 pino.logger.info(`Helia is running! PeerID: ${helia.libp2p.peerId.toString()}`)
 
 autoPeering.start()
+diskUsageCron.start()
 
 const PORT = config.serverPort
 const app = express()
 
 app.use(pino)
 
+const corsOptions = {
+  origin: config.cors.originRegexps.map((item: string) => new RegExp(`^https?:\\/\\/${item}`)),
+  methods: ['GET', 'POST']
+}
+app.use(cors(corsOptions))
```

Comment:
The `corsOptions` is not typed. Pass the object directly to `cors({...})`.

![image](https://github.com/Adamant-im/ipfs-node/assets/25831507/10661298-3886-471c-aa26-a32eb603b9d2)